pending-documentation.md

```diff
@@ -276,16 +277,193 @@ syntax Pgm ::= underbar(NameWithUnderbar)
+not a unique parse for that  term. In this case, you can get all possible
+parses that were valid by means of the `--gen-glr-parser` command line flag.
+Usually, however, this indicates a problem in your grammar.
+
+Priority is specified in K grammars by means of one of two different
+mechanisms. The first, and simplest, simply replaces the `|` operator in a 
+sequence of K productions with the `>` operator. This operator indicates that
+everything prior to the `>` operator (including transitively) binds tighter
+than what comes after. For example, a more complete grammar for simple
+arithmetic might be:
+
+```k
+syntax Exp ::= Exp "*" Exp
+             | Exp "/" Exp
+             > Exp "+" Exp
+             | Exp "-" Exp
+```
+
+This indicates that multiplication and division bind tigher than addition
+and subtraction, but that there is no relationship in priority between
+multiplication and division.
+
+As you may have noticed, this grammar is also ambiguous. `x*y/z` might refer to
+`x*(y/z)` or to `(x*y)/z`. Indeed, if we removed division and subtraction
+entirely, the grammar would still be ambiguous: `x*y*z` might parse as
+`x*(y*z)`, or as `(x*y)*z`. To resolve this, we introduce another feature:
+associativity. Roughly, asssociativity tells us how symbols are allowed to nest
+within other symbols with the same priority. If a set of symbols is left
+associative, then symbols in that set cannot appear as the rightmost child
+of other symbols in that set. If a set of symbols is right associative, then
+symbols in that set cannot appear as the leftmost child of other symbols in
+that set. Finally, if a set of symbols is non-associative, then symbols
+in that set cannot appear as the rightmost or leftmost child of other symbols
+in that set. For example, in the above example, if addition and subtraction
+are left associative, then `x+y+z` will parse as `(x+y)+`z and `x+y-z` will
+parse as `(x+y)-z` (because the other parse will have been rejected).
+
+You might notice that this seems to apply only to binary infix operators. In
+fact, the real behavior is slightly more complicated. Priority and
+associativity (for technical reasons that go beyond the scope of this document)
+really only apply when the rightmost or leftmost item in a production is a
+nonterminal. If the rightmost nonterminal is followed by a terminal (or
+respectively the leftmost preceded), priority and associativity do not apply.
+Thus we can generalize these concepts to arbitrary context-free grammars.
+
+Associativity is specified in K grammars by means of one of two different
+mechanisms. The first, and simplest, adds the associativity of a priority block
+of symbols prior to that block. For example, we can remove the remaining
+ambiguities in the above grammar like so:
+
+```k
+syntax Exp ::= left:
+               Exp "*" Exp
+             | Exp "/" Exp
+             > left:
+               Exp "+" Exp
+             | Exp "-" Exp
+```
```

Comment:
I would have one of these examples be `left`, and the other `right`, to show that you can use both.